index.html

```diff
@@ -0,0 +1,268 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js" defer></script>
+  <title>Abstract</title>
+</head>
+<body>
+  <div class="container">
+    <!-- ######################################### SCROLL TO TOP ######################################## -->
+    <button onclick="scrollTopFunction()" id="scrollToTopButton" title="Go to top"><img src="images/arrows.png" class="arrows"></button>
+    <!-- ######################################### HEADER ######################################### -->
+    <div class="header" id="header">
+      <div id="header_logo_dark">
+        <img src="images/logo-2.svg" id="header_logo_dark">
+      </div>
+      <div id="header_logo">
+        <img src="images/logo.svg" id="header_logo">
+      </div>
+      <div class="header_item digital">
+        Digital Agency
+        <img src="images/down-arrow.png" alt="arrow" class="header_item_arrow">
+      </div>
+      <div class="header_item">
+        Services
+        <img src="images/down-arrow.png" alt="arrow" class="header_item_arrow">
+      </div>
+      <div class="header_item">
+        Portfolio
+        <img src="images/down-arrow.png" alt="arrow" class="header_item_arrow">
+      </div>
+      <div class="header_item">
+        Pages
+        <img src="images/down-arrow.png" alt="arrow" class="header_item_arrow">
+      </div>
+      <div class="header_item">
+        Blog
+        <img src="images/down-arrow.png" alt="arrow" class="header_item_arrow">
+      </div>
+      <div class="header_item">
+        Contact
+      </div>
+      <div class="header_menu">
+        <img src="images/menu.png" alt="manu-icon" class="header_menu_icon">
+      </div>
+    </div>
+    <!-- ######################################### DARK MODE ######################################### -->
+    <div class="darkmode"><img src="images/moon.png" alt="moon-icon" class="darkmode_icon"></div>
+    <!-- ######################################### BANER SECTION ######################################### -->
+    <div class="container_top">
+      <div class="container_top_background">
+        <img src="images/bubble-21.png" class="bubble21">
+        <img src="images/banner-thumb-4.png" class="banner">
+        <img src="images/line-4.png" class="line4">
+        <img src="images/bubble-9.png" class="bubble9">
+      </div>
+      <div class="container_top_conent">
+        <div class="baner_text">
+          <div class="baner_title">Design</div>
+          It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum.
```

Comment:
Jak dłuższy tekst to bym dał jako paragraf, tylko trzeba sprawdzić czy znacznik `<p></p>` nie doda nam marginesów (zależy od tego co jest w normalizacji stylów).

```suggestion
          <p>It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum.</p>
```

To samo w innych miejscach.